content/_index.md

```diff
@@ -48,6 +48,7 @@ Ready to get started? [Click here](getting-started).
 			</div>
 			<div class="col col-auto">
 				<button class="btn btn-secondary playground-btn-flash" disabled>Download binary</button>
+				<a href="/tour/" class="btn btn-link">Tour of TinyGo</a>
```

Comment:
```suggestion
                               {{< button href="/tour/" class="btn btn-secondary" >}}Tour of TinyGo{{< /button >}}
```

Reply:
Are you sure about that?

```
Error: error building site: process: readAndProcessContent: "/home/ayke/src/tinygo/site/content/_index.md:31:1": failed to extract shortcode: template for shortcode "button" not found
```

I prefer a link-style button: to me, buttons are something you do on a given page (like in a webapp for example), and links lead to a different page. This one clearly goes to a different page.

Reply:
Let's just go with what you have now, and then we can change later if we thing we need to.